test/elm/datetime/datetime-test.ts

```diff
@@ -1064,6 +1064,22 @@ describe('DurationBetween', () => {
   it('should properly execute hours between when springing ahead for DST', async function () {
     (await this.hoursBetween1and3CrossingFallDST.exec(this.ctx)).should.equal(3);
   });
+
+  it('should not return an uncertainty when the DateTimes have 0 ms', async function () {
+    (await this.durationInDaysWith0MS.exec(this.ctx)).should.equal(1);
+  });
+
+  it('should not return an uncertainty when the DateTimes do not specify ms', async function () {
+    (await this.durationInDaysNoMS.exec(this.ctx)).should.equal(1);
+  });
+
+  it('should not return an uncertainty when the DateTimes have the same nonzero ms', async function () {
+    (await this.durationInDaysSameMS.exec(this.ctx)).should.equal(1);
+  });
+
+  it('should not return an uncertainty when the DateTimes have different nonzero ms', async function () {
+    (await this.durationInDaysDiffMS.exec(this.ctx)).should.equal(1);
+  });
```

Comment:
Once you fix the CQL test (as noted in a previous comment), I think the expected answer here will be `0` because it's measuring duration from MS 59 to MS 58 -- which comes short of a day by 1 MS.

Reply:
Addressed in [a638a5f](https://github.com/cqframework/cql-execution/pull/340/commits/a638a5f0f197965e941a09d7740343001ce038c9)